RocketChat/Rocket.Chat.ReactNative
# Chore: Migrate notification/push to Typescript

## Proposed changes


Migrate the notification/push files to ts.
It was necessary to remove the extension from the push.android.ts file to solve the typescript import, however as the bundle understands that the no extension is for use on android and .ios is for use on ios, I don't think there is any problem.


## Issue(s)	


## How to test or reproduce


receive and click on a notification (app on background and foreground)
Verify badge count on ios

## Screenshots

## Types of changes




## Checklist



## Further comments
